xsensmt/XsensMT.cpp

```diff
@@ -364,6 +364,18 @@ bool XsensMT::open(yarp::os::Searchable &config)
         return false;
     }
 
+    if(config.check("reset"))
+    {
+        bool okReset = m_xsensDevice->resetOrientation(XRM_Alignment);
+        if(okReset)
+        {
+            yDebug() << "the reset was successful";
```

Comment:
It's better to add a prefix like this line
https://github.com/robotology/yarp-device-xsensmt/blob/7a2de59ad2343be23fce86db850215a4f18d9e3e/xsensmt/XsensMT.cpp#L380
Because this device might be running with many other devices.

Also being more verbose here to explain what exactly is being reset can be helpful for the user.